.github/workflows/prod-build.yml

```diff
@@ -155,6 +155,9 @@ jobs:
           # exception in the world where we actually want to welcome robots.
           BUILD_ALWAYS_ALLOW_ROBOTS: true
 
+          # Browser-compat data.
+          REACT_APP_BCD_BASE_URL: https://bcd.developer.mozilla.org
```

Comment:
Needs accompanying documentation in `docs/envvars.md` - there you might want to document what to set this to if you want to use the local version of bcd, rather than whatever we have uploaded to allizom